proposals/lip-0038.md

```diff
@@ -323,20 +329,20 @@ PopVerify(pk, proof) = INVALID
 ```python
```

Comment:
For the test vectors, we assume that the functions `Verify` and `FastAggregateVerify` return either `VALID` or `INVALID`, which contradicts the specifications above. We could change it here or update the function code above to something like
```
return FastAggregateVerify(keys, taggedMessage, signature) == VALID
```

Reply:
Moreover, in the test vectors we always use `=` instead of `==` in the last line.